src/api.ts

```diff
@@ -80,6 +81,9 @@ export interface StudioBlueprintManifest extends BlueprintManifestBase {
 
 	/** Returns information about the playlist this rundown is a part of, return null to not make it a part of a playlist */
 	getRundownPlaylistInfo?: (rundowns: IBlueprintRundownDB[]) => BlueprintResultRundownPlaylist | null
+
+	/** Process config before storing it by core to later be returned by context's getShowStyleConfig */
+	parseConfig?: (config: IBlueprintConfig) => unknown
```

Comment:
This is a minor thing, and possibly personal opinion, but the naming `parseConfig` sounds like it's imperative, ie a function. I'd prefer naming it `parsedConfig`

Reply:
also, could we add in the description what happens if this is omitted? Does it default to the old behaviour?

Reply:
perhaps `preprocessConfig` would be a better name? 

> sounds like it's imperative, ie a function

but it is a function, so I dont follow

Reply:
Agreed, `preprocessConfig` is even better.

>sounds like it's imperative, ie a function
>>but it is a function, so I dont follow

me neither, apparently..

Reply:
Renamed to `preprocessConfig`.